swagger/pe-openapi-0.0.1.yaml

```diff
@@ -1,30 +1,47 @@
 openapi: 3.0.3
 info:
   description: Decentralized Identity Foundation Presentation Exchange API
-  version: "1.0.0"
+  version: '1.0.0'
   title: DIF PE API
   license:
     name: Apache 2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+
+servers:
```

Comment:
How do we run Swagger-UI?

Reply:
We never tried to run that. But We can indeed run the commands mentioned in the README (with some alterations) to see a file made that should provide all the end point implemations stubs.

Reply:
clean install -P api-typescript

Reply:
pushed the readme.